src/unix/fs.c

```diff
@@ -231,7 +231,29 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 
 
 static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
-  return mkdtemp((char*) req->path) ? 0 : -1;
+  char *path = (char*) req->path;
+
+#if defined(__MVS__)
+  /* There is no mkdtemp. So instead use mktemp to generate a
```

Comment:
What would the alternative be?
The windows implementation (afaik) we generate a random string and attempt to make a directory with it, if it already exists we try again for `TMP_MAX` amount of times or until we succeed.
I guess one of the issues of using `mkstemp` and then removing the file is I am not sure whether or not there is a guarantee that nothing else creates a file with the same name in time between the `remove`and `mkdir`